runtime-testsuite/test/org/antlr/v4/test/runtime/go/GoRunner.java

```diff
@@ -111,36 +114,61 @@ protected String grammarParseRuleToRecognizerName(String startRuleName) {
 		return rn;
 	}
 
+	@Override
+	protected  List<String> getTargetToolOptions() {
+		ArrayList<String> options = new ArrayList<String>();
+		options.add("-o");
+		options.add(tempTestDir.resolve("parser").toString());
+		return options;
```

Comment:
It can be calculated once and used for every test since `tempTestDir` is calculated once.

Reply:
I have cached it now.